playbooks/templates/.env.j2

```diff
@@ -268,6 +268,19 @@ ABUSE_LOG_LEVEL={{ webportal_common_config.abuse_log_level }}
 # REQUIRED: Hardcoded, not configurable
 ABUSE_SPONSOR="Abuse Scanner"
 
+# Reporting related environment variables
+#
+# OPTIONAL: manually defined by user, defaults to false
+ABUSE_NCMEC_REPORTING_ENABLED={{
+webportal_common_config.abuse_ncmec_reporting_enabled | default('false') }}
+#
+# REQUIRED: manually defined by user, only if reporting is enabled
+{% if webportal_common_config.abuse_ncmec_reporting_enabled %}
+NCMEC_USERNAME={{ webportal_common_config.ncmec_username }}
+NCMEC_PASSWORD={{ webportal_common_config.ncmec_password }}
+NCMEC_DEBUG={{ webportal_common_config.ncmec_debug | default('true') }}
+{% endif %}
```

Comment:
See other `{% if %}` blocks how we handle whitespace (we don't want to add double empty new lines), we enclose trainilng new line inside the block before `{% endif %}`.